bitnami/concourse/templates/config/secret-external-db.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if and (not .Values.postgresql.enabled) (not .Values.externalDatabase.existingSecret) (not .Values.postgresql.existingSecret)}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ printf "%s-%s" .Release.Name "externaldb"  }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+      {{- include "common.tplvalues.render" (dict "value" .Values.commonLabels "context" $) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" (dict "value" .Values.commonAnnotations "context" $) | nindent 4 }}
+  {{- end }}
+type: Opaque
+data:
+  {{ include "concourse.database.existingsecret.key" . }}: {{ .Values.externalDatabase.password | b64enc | quote }}
```

Comment:
I actually don't see much value in allowing a custom key if we are creating the secret

Reply:
Use `postgresql-password` here. It actually doesn't make much sense unless you are using the externalDatabase.existingSecret value